modules/generators/random/random_word/secgen_local/local.rb

```diff
@@ -8,16 +8,7 @@ def initialize
   end
 
   def generate
-    # require 'wordlist'
-    #
-    # list = Wordlist::FlatFile.new("#{ROOT_DIR}/lib/resources/wordlists/wordlist")
-    # list.each_unique do |word|
-    #   outputs << word
-    #   break
-    # end
-
-    self.outputs << File.readlines("#{ROOT_DIR}/lib/resources/wordlists/wordlist").sample.chomp
-
+    self.outputs << File.readlines("#{WORDLISTS_PATH}/wordlist").sample.chomp
```

Comment:
This is better. But can we call it WORDLISTS_DIR to be more consistent? (Possibly updating any other "PATH" constants also)